detect/git.go

```diff
@@ -48,9 +49,24 @@ func (d *Detector) DetectGit(gitCmd *sources.GitCmd) ([]report.Finding, error) {
 						FilePath:  gitdiffFile.NewName,
 					}
 
+					var timer *time.Timer
+					if d.SlowWarningThreshold.Seconds() > 0 {
+						timer = time.AfterFunc(d.SlowWarningThreshold, func() {
+							e := log.Warn()
+							if commitSHA != "" && len(commitSHA) > 7 {
+								e.Str("commit", commitSHA[:7])
+							}
```

Comment:
This probably isn't necessary. I thought both Git and non-Git were handled here.